.vscode/settings.json

```diff
@@ -11,5 +11,6 @@
   },
 
   "typescript.tsdk": "node_modules/typescript/lib",
-  "typescript.enablePromptUseWorkspaceTsdk": true
+  "typescript.enablePromptUseWorkspaceTsdk": true,
+  "typescript.preferences.importModuleSpecifier": "relative"
```

Comment:
For some reason, VSCode started to generate absolute imports... So I'm forcing TypeScript to generate relative imports instead.